static/app/constants/index.tsx

```diff
@@ -553,6 +553,34 @@ export const DATA_CATEGORY_INFO = {
       showExternalStats: true,
     },
   },
+  [DataCategoryExact.PREVENT_USER]: {
+    name: DataCategoryExact.PREVENT_USER,
+    plural: DataCategory.PREVENT_USER,
+    singular: 'preventUser',
+    displayName: 'prevent user',
+    titleName: t('Prevent Users'),
+    productName: t('Prevent Users'),
+    uid: 29,
+    isBilledCategory: true,
+    statsInfo: {
+      ...DEFAULT_STATS_INFO,
+      showExternalStats: true,
```

Comment:
this will render it in the stats control automatically unless you add a filter where `CHART_OPTIONS_DATACATEGORY` is used (i'd recommend flipping this when you do stats page stuff instead)